content/projects/gsoc/2023/project-ideas/docker-compose-build.adoc

```diff
@@ -45,7 +45,7 @@ Of course, these snippets could also be run thanks to link:https://www.gitpod.io
 * link:https://docs.docker.com/compose/compose-file/compose-file-v3/[Docker Compose file format v3]
 * link:https://www.jenkins.io/projects/jcasc/[Jenkins Configuration as Code]
 * link:https://plugins.jenkins.io/job-dsl/[Job DSL]
-* link:/book/installing/kubernetes/[Kubernetes]
+* link:https://www.jenkins.io/doc/book/installing/kubernetes/[Kubernetes]
```

Comment:
```suggestion
* link:/doc/book/installing/kubernetes/[Kubernetes]
```

Reply:
Then I think we might as well update L46 to be consistent. What do you think @zbynek?

Reply:
> Then I think we might as well update L46 to be consistent. What do you think @zbynek?

Off-topic, let's keep PR feedback on the changes proposed, not on general items, that _could_ be addressed.

Reply:
Okay, I can see your point @NotMyFault though it could be confusing to the code contributor since they may be wondering why there are two different ways to link the page within the same context.

Reply:
Works either way.